dayton/spiders/reap_spider.py

```diff
@@ -56,11 +58,19 @@ def getTaxEligibility(self, response):
     def getPaymentPlan(self, response):
         sel = Selector(response)
         item = response.meta['item']
-        item['paymentplan'] = False
-        for paymentplan in sel.xpath('//*[@id="content_middle_wide"]/div/form/div/tr/td/font/font/b/text()').extract():
-            item['paymentplan'] = 'Unapplied Payments:' in paymentplan or item['paymentplan']
+        item['payment_plan'] = False
+        
+        for paymentXPath in sel.xpath('//*[@id="content_middle_wide"]/div[3]/form/table/tr'):
+            try:
```

Comment:
I get an error about an unexpected unindent. I can fix it by removing the `try: ` line. Should this have an `except` block?